### PR TITLE
Some minor changes

### PR DESCRIPTION
**Pull request by ritaturk**
Changed composed to comprised, changed useless to non-essential, changed Xamarin link to open up on new page,

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/SiliconStudio/xenko-docs/pull/27" target="_blank">ReviewNinja

File: manual/get-started/create-a-project.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To create an empty game:
 5. **Select the platform(s)** you would like your game to support from the *Platforms* section.  
     If your development system does not have the required prerequisites installed for any of the selected platform, a warning message is displayed.
     > [!Note]
-    >  To run your game on iOS and Android, you need to install [Xamarin](https://www.xamarin.com/studio) (free if you have Visual Studio).
+    >  To run your game on iOS and Android, you need to install [<a href=https://www.xamarin.com/studio target="_blank"> Xamarin</a>] (free if you have Visual Studio).
```
Comment (Kryptos-FR) [inline manual/get-started/create-a-project.md]: The brackets [ and ] should be removed if you use a HTML tag.

File: manual/get-started/create-a-project.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ To create an empty game:
 5. **Select the platform(s)** you would like your game to support from the *Platforms* section.  
     If your development system does not have the required prerequisites installed for any of the selected platform, a warning message is displayed.
     > [!Note]
-    >  To run your game on iOS and Android, you need to install [Xamarin](https://www.xamarin.com/studio) (free if you have Visual Studio).
+    >  To run your game on iOS and Android, you need to install <a href=https://www.xamarin.com/studio target="_blank"> Xamarin</a> (free if you have Visual Studio).
```
Comment (manu-silicon) [inline manual/get-started/create-a-project.md]: To be correct, you also need the double quotes around the URL.

Comment (ritaturk) [inline manual/get-started/create-a-project.md]: okay.

**Comment by manu-silicon** [comment]
Thanks!

**Pull request merged**